commands/qemu-guest-tools/actions_test.go

```diff
@@ -72,7 +72,7 @@ func TestGuestToolsProcessingActions(t *testing.T) {
 	defer g.StopProcessingActions()
 
 	////////////////////
-	debug("### Test meta.GetArtifact")
+	t.Log("### Test meta.GetArtifact")
```

Comment:
Basically, if we don't use `debug` these messages will arrive out of order... Which is bad when using it for debugging...